src/lib.rs

```diff
@@ -83,7 +83,7 @@ impl MemFile {
 	///
 	/// See [`Self::create`] for more information.
 	pub fn create_default(name: &str) -> std::io::Result<Self> {
-		Self::create(name, &CreateOptions::default())
+		Self::create(name, CreateOptions::default())
 	}
```

Comment:
Why not impl Default ?

Reply:
Oh never mind the name... I guess a empty "" name is ok but meh.

Reply:
a `CreateOptions::default()::create("toto")` is already enough IMO we don't need this.

Reply:
For `MemFile`? Because `create` returns an `std::io::Result`.

Reply:
> a `CreateOptions::default()::create("toto")` is already enough IMO we don't need this.

Maybe, but I prefer to write `MemFile::create_default("toto")`. I find it more readable.